api/task_manager.cc

```diff
@@ -81,6 +84,11 @@ tm::task_status make_status(full_task_status status) {
     res.progress_units = status.task_status.progress_units;
     res.progress_total = status.progress.total;
     res.progress_completed = status.progress.completed;
+    std::vector<std::string> ct{status.child_tasks.size()};
+    boost::transform(status.child_tasks, ct.begin(), [] (const auto& child) {
+        return child->id().to_sstring();
+    });
```

Comment:
This conversion should happen earlier and here the id's should be returned as strings in the json result